skunkworks_crow/build.gradle

```diff
@@ -6,7 +6,7 @@ android {
     defaultConfig {
         applicationId "org.odk.share"
         minSdkVersion 16
-        targetSdkVersion 22
+        targetSdkVersion 27
```

Comment:
Please revert this unrelated change. This should be fixed separately as increasing the target version will have more changes too